README.md

```diff
@@ -184,6 +184,12 @@ which is the front-end pattern library used in this project.
 
 ## Working with the templates
 <!-- Perhaps we want to split this out into a separate page? -->
+### Front-End Template/Asset Locations ###
+
+**Templates** that are served by the django server: ```cfgov\v1\jinja2\v1 ```
```

Comment:
Trivial and can get picked up in a cleanup PR, but three back-ticks is for multi-line comments. You only need single back-ticks for inline code snippets https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting